.github/workflows/chaos.yaml

```diff
@@ -54,7 +54,7 @@ jobs:
       uses: chaos-mesh/chaos-mesh-actions@master
       env:
         CFG_BASE64: YXBpVmVyc2lvbjogY2hhb3MtbWVzaC5vcmcvdjFhbHBoYTEKa2luZDogUG9kQ2hhb3MKbWV0YWRhdGE6CiAgbmFtZTogcG9kLWZhaWx1cmUKICBuYW1lc3BhY2U6IHBpbmdjYXAKc3BlYzoKICBhY3Rpb246IHBvZC1mYWlsdXJlIAogIG1vZGU6IG9uZSAKICBkdXJhdGlvbjogIjYwcyIKICBzZWxlY3RvcjoKICAgIGxhYmVsU2VsZWN0b3JzOgogICAgICAiYXBwLmt1YmVybmV0ZXMuaW8vY29tcG9uZW50IjogImNvbnRyb2xsZXItbWFuYWdlciIKICBzY2hlZHVsZXI6CiAgICBjcm9uOiAiQGV2ZXJ5IDVtIgo=
-        CHAOS_MESH_VERSION: v1.0.0
+        CHAOS_MESH_VERSION: v2.1.0
```

Comment:
Seems apart from bumping chaos-mesh version, we also need to modify the `CFG_BASE64` spec above. In chaos-mesh v2, `scheduler` field is deprecated (details in https://github.com/chaos-mesh/chaos-mesh/pull/1578, https://github.com/chaos-mesh/rfcs/pull/15, https://github.com/chaos-mesh/chaos-mesh/discussions/1520), and it seems the chaos-mesh team provided some docs for migration: https://chaos-mesh-website-archived.netlify.app/docs/upgrade-to-2.0/

Reply:
According to the doc, the chaos spec should be modified to something like
```yaml
apiVersion: chaos-mesh.org/v1alpha1
kind: Schedule
metadata:
  name: pod-failure
  namespace: pingcap
spec:
  schedule: '@every 5m'
  type: PodChaos
  historyLimit: 5
  concurrencyPolicy: Forbid
  podChaos:
    action: pod-failure
    mode: one
    duration: 60s
    selector:
      labelSelectors:
        app.kubernetes.io/component: controller-manager
```